views/deductions/trillium-start.pug

```diff
@@ -0,0 +1,16 @@
+extends ../base
+
+block variables
+  -var title = __('The Ontario Trillium Benefit')
+  -var trilliumRentClaim = !hasData(data, 'deductions.trilliumRentClaim') ? '' : data.deductions.trilliumRentClaim ? 'Yes' : 'No'
+
+block content
+
+  h1 #{title}
+
+  div
+    p #{__('The Ontario Trillium Benefit (OTB) is a government payment that helps with energy, sales and property taxes. You can learn about the OTB at')} <a href='https://www.canada.ca/en/revenue-agency/services/child-family-benefits/provincial-territorial-programs/ontario-trillium-benefit-questions-answers.html'>https://www.canada.ca/en/revenue-agency/services/child-family-benefits/provincial-territorial-programs/ontario-trillium-benefit-questions-answers.html</a>
```

Comment:
the way we do links is that we want to put the actual `href` in as a translateable string as well, like we do here: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/views/_includes/offramp-bullets.pug#L2

That way, when we switch the page to french, the url also changes.